bigtable/docs/snippets.py

```diff
@@ -160,6 +171,188 @@ def test_bigtable_create_app_profile():
     app_profile.delete(ignore_warnings=True)
 
 
+def test_bigtable_create_table():
+    # [START bigtable_create_table]
+    from google.cloud.bigtable import Client
+    from google.cloud.bigtable import column_family
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+
+    # Create table without Column families.
+    table1 = instance.table("table_id1")
+    table1.create()
+
+    # Create table with Column families.
+    table2 = instance.table("table_id2")
+    # Define the GC policy to retain only the most recent 2 versions.
+    max_versions_rule = column_family.MaxVersionsGCRule(2)
+    table2.create(column_families={'cf1': max_versions_rule})
+
+    # [END bigtable_create_table]
+    assert table1.exists()
+    assert table2.exists()
+    table1.delete()
+    table2.delete()
+
+
+def test_bigtable_sample_row_keys():
+    # [START bigtable_sample_row_keys]
+    from google.cloud.bigtable import Client
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+
+    table = instance.table("table_id1_samplerow")
+    # [END bigtable_sample_row_keys]
+    initial_split_keys = [b'split_key_1', b'split_key_10',
+                          b'split_key_20']
+    table.create(initial_split_keys=initial_split_keys)
+    # [START bigtable_sample_row_keys]
+    data = table.sample_row_keys()
+    actual_keys, offset = zip(*[(rk.row_key, rk.offset_bytes) for rk in data])
+    # [END bigtable_sample_row_keys]
+    initial_split_keys.append(b'')
+    assert actual_keys == initial_split_keys
```

Comment:
Ether cast `actual_keys` to `list()` or `initial_split_keys` to `tuple()` please.

Reply:
Unable to get any data on emulators. My rpc calls are broken so not able to test on cloud.